views/includes/csrf_token.html

```diff
@@ -0,0 +1,7 @@
+{% from "web-security-node/components/csrf-token-input/macro.njk" import csrfTokenInput %}
+
+{{
+    csrfTokenInput({
+    csrfToken: csrfToken
```

Comment:
line 5 needs indenting

Reply:
I have problems with indentation in the templates when I save the file. I will try to set up my ide to avoid this.